src/instructlab/model/backends/backends.py

```diff
@@ -304,6 +309,100 @@ def shutdown_process(process: subprocess.Popen, timeout: int) -> None:
         )
         process.kill()
 
+    # Various facilities of InstructLab rely on multiple successive start/stop
+    # cycles. Since vLLM relies on stable VRAM for estimating capacity, residual
+    # reclamation activity can lead to crashes on start. To prevent this add a
+    # short delay (typically ~ 10 seconds, max 30) to verify stability.
+    #
+    # Ideally a future enhancement would be contributed to vLLM to more gracefully
+    # handle this condition.
+    wait_for_stable_vram(MAX_STABLE_VRAM_WAIT)
+
+
+def wait_for_stable_vram(timeout: int):
```

Comment:
is it `int` though?.. Where do you convert it to `int`? It's a string, it seems.